src/App.tsx

```diff
@@ -19,55 +63,76 @@ export const App: React.FC = () => {
-
-              <div className="dropdown-item" data-cy="suggestion-item">
-                <p className="has-text-link">Pieter Bernard Haverbeke</p>
-              </div>
-
-              <div className="dropdown-item" data-cy="suggestion-item">
-                <p className="has-text-link">Pieter Antone Haverbeke</p>
-              </div>
-
-              <div className="dropdown-item" data-cy="suggestion-item">
-                <p className="has-text-danger">Elisabeth Haverbeke</p>
-              </div>
-
-              <div className="dropdown-item" data-cy="suggestion-item">
-                <p className="has-text-link">Pieter de Decker</p>
-              </div>
-
-              <div className="dropdown-item" data-cy="suggestion-item">
-                <p className="has-text-danger">Petronella de Decker</p>
-              </div>
+          <div className="control">
+            <button className="button is-info" onClick={handlerClear}>
+              Clear
+            </button>
+          </div>
 
-              <div className="dropdown-item" data-cy="suggestion-item">
-                <p className="has-text-danger">Elisabeth Hercke</p>
-              </div>
-            </div>
+          <div
+            className="dropdown-menu custom-dropdown-menu"
+            role="menu"
+            data-cy="suggestions-list"
+          >
+            {isDropedDown &&
+              getFilterPeople().map(person => (
+                <div
+                  className="
+                  dropdown-content
+                  custom-dropdown-item
+                  "
+                  key={person.name}
+                  onClick={() => {
+                    setQuery(person.name);
+                    setIsDropedDown(false);
+                    setSelectedPerson(person);
+                  }}
+                  style={{
+                    cursor: 'pointer',
+                  }}
+                >
+                  <div className="dropdown-item" data-cy="suggestion-item">
+                    <p className="has-text-link">{person.name}</p>
+                  </div>
+                </div>
+              ))}
           </div>
         </div>
 
-        <div
-          className="
-            notification
-            is-danger
-            is-light
-            mt-3
-            is-align-self-flex-start
-          "
-          role="alert"
-          data-cy="no-suggestions-message"
-        >
-          <p className="has-text-danger">No matching suggestions</p>
-        </div>
+        {getFilterPeople().length === 0 ? (
+          <div
+            className="
+              notification
+              is-danger
+              is-light
+              mt-3
+              is-align-self-flex-start"
+            role="alert"
+            data-cy="no-suggestions-message"
+          >
+            <p className="has-text-danger">No matching suggestions</p>
+          </div>
+        ) : selectedPerson && getFilterPeople().length > 0 ? (
```

Comment:
Avoid using nested ternary

Reply:
sorry, but this is not nested ternary, i use the only one ternary expression that compares two conditions via the operator ?: to determine which JSX block to render

Reply:
yes it's actually called nested ternary, try to rewrite without this antipattern